cluster-api/cluster-api/go.mod

```diff
@@ -2,7 +2,7 @@ module openshift/installer/cluster-api/providers/core
 
 go 1.21
 
-require sigs.k8s.io/cluster-api v1.7.0
+require sigs.k8s.io/cluster-api v1.7.2
```

Comment:
Nice. Let's just make sure we run tests for all capi providers to check this doesn't cause issues.